cmd/bb_storage/main.go

```diff
@@ -24,9 +25,17 @@ import (
 
 func main() {
 	if len(os.Args) != 2 {
-		log.Fatal("Usage: bb_storage bb_storage.jsonnet")
+		log.Fatal("Usage: bb_storage [--dump_configuration_struct] bb_storage.jsonnet")
```

Comment:
Nit: UNIX usage/man page synopses always use `[]` to indicate optionality and `|` to indicate choice. The correct usage would therefore be:
```
Usage: bb_storage --dump_configuration_struct | filename
```

Reply:
Trueee, I was just checking your historical unix backgroud. You passed. =) good catch.